docs/helm/hydra.md

```diff
@@ -229,3 +229,12 @@ $ hydra token client \
     --client-id my-client \
     --client-secret secret
 ```
+
+### Hydra-maester
+This chart includes a helper chart in the form of [Hydra-maester](https://github.com/ory/k8s/blob/master/docs/helm/hydra-maester.md), a K8s controller, which manages OAuth2 clients using the `oauth2clients.hydra.ory.sh` custom resource. This component is enabled by default, and installed together with Hydra, however it can be disabled by setting the proper flag:
```

Comment:
```suggestion
This chart includes a helper chart in the form of [Hydra Maester](https://github.com/ory/k8s/blob/master/docs/helm/hydra-maester.md), a Kubernetes controller, which manages OAuth2 clients using the `oauth2clients.hydra.ory.sh` custom resource. By default, this component is enabled and installed together with Hydra. However it can be disabled by setting the proper flag:
```